src/main/java/by/home/controller/DataReader.java

```diff
@@ -0,0 +1,47 @@
+package by.home.controller;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataReader {
+    public List<String> read(String FilePath) {
+
+        BufferedReader bufferedReader = null;
+        List<String> lines = null;
+
+        try {
+
+            FileReader fr = new FileReader(FilePath);
+            bufferedReader = new BufferedReader(fr);
+
+            lines = new ArrayList<String>();
+
+            String line;
+
+            while ((line = bufferedReader.readLine()) != null) {
```

Comment:
try not to call method inside while ( )
butter read one line before while and at the end of the loop